dist/index.html

```diff
@@ -1,42 +1,28 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <script src="https://kit.fontawesome.com/fe6dec7cca.js" crossorigin="anonymous"></script>
-    <title>My To-do List</title>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet" crossorigin="anonymous">
+    <script src="https://use.fontawesome.com/60c3c5e662.js" crossorigin="anonymous"></script>
+    <title>Tobin's Todo Li</title>
 <script defer src="main.js"></script></head>
-
 <body>
-    <div class="list">
-        <div class="row title">
-            <p>Today's To Do</p>
-            <i class="fas fa-sync"></i>
-        </div>
-        <div class="row">
-            <input class="add-input" type="text" placeholder="Add to your list">
-            <i class="fas fa-level-down-alt rotate"></i>
-        </div>
-        <div class="row">
-            <input type="checkbox">
-            <p>A task to be done</p>
-            <i class="fas fa-ellipsis-v"></i>
-        </div>
-        <div class="row">
-            <input type="checkbox">
-            <p>A task to be done</p>
-            <i class="fas fa-ellipsis-v"></i>
-        </div>
-        <div class="row">
-            <input type="checkbox">
-            <p>A task to be done</p>
-            <i class="fas fa-ellipsis-v"></i>
-        </div>
-        <div class="row clear">
-            <p>Clear All Completed</p>
-        </div>
+  <section class="container">
+    <div class="header d-flex">
+      <h1>Today's To-Do</h1>
+      <i class="fa fa-refresh"></i>
     </div>
+    <form id="form" class="d-flex">
+      <input type="text" id="input" name="new_task" placeholder="Add to your list...">
+      <button class="fa fa-level-down fa-rotate-90" type="submit" title="submit"></button>
+    </form>
+    <div class="task-container">
+      <ul class="task-list">
+      </ul>
+    </div>
+    <span id="clear-completed"> Clear all Completed</span>
```

Comment:
- [ ] To center this text vertically kindly a padding-bottom to this ☝️  `span`.